iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/MultiVariantTopLevelObject.java

```diff
@@ -10,10 +10,10 @@
  *
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-interface MultiStateObject {
+interface MultiVariantTopLevelObject {
```

Comment:
This name is not very clear: what about something like `VariantManagerHolder`

Reply:
Done